### PR TITLE
CHANGES MADE

### PR DESCRIPTION
**Pull request by bhaveshgoyal27**

### CODE REVIEW

**Review by YashMeh — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,46 @@
...
+```bash
+pip install selenium
+pip install selenium-requests
+```
+
+Use the link [geckodriver](https://github.com/mozilla/geckodriver/releases) to install the webdriver.
+
+## Usage
+
+Clone the repo and in the edit the people.csv in the Samples folder as your need
```
Comment (YashMeh) [inline README.md:18]: Grammatical error.

**Review by YashMeh — CHANGES_REQUESTED** [review]
Instead of asking the user to make changes directly to the script- 
- Ask for user input when user launches index.py.
- Convert the whatsapp_group.py code into a single function call.
- Pass those parameters to the function.

File: README.md [hunk 2]
```diff
@@ -1 +1,46 @@
-# whatsapp-groupadd
\ No newline at end of file
+# Whatsapp-groupadd
+
+Whatsapp-groupadd is a Python code which helps it users to add people to a specific whatsapp group without directly saving their numbers.
+
+## Installation
+
```
Comment (vinitshahdeo) [inline README.md:6]: Mention `requirements` too. 
It's good to have pre-requisites before Installation steps.

File: README.md [hunk 3]
```diff
@@ -1 +1,46 @@
...
+![alt text](https://imgur.com/QtZuyKh)
+
+Click on Add Participant
+
+## Contributing
+Pull requests are welcome. For major changes, please open an issue first to discuss what you would like to change.
+
+Please make sure to update tests as appropriate.
+
```
Comment (vinitshahdeo) [inline README.md:46]: Do we also have tests?

File: README.md [hunk 4]
```diff
@@ -1 +1,46 @@
-# whatsapp-groupadd
\ No newline at end of file
+# Whatsapp-groupadd
+
+Whatsapp-groupadd is a Python code which helps it users to add people to a specific whatsapp group without directly saving their numbers.
+
```
Comment (vinitshahdeo) [inline README.md:4]: **WhatsApp** is a proper noun. It should be capitalized always.
Also, it looks odd when you write _whatsapp, Whatsapp_ etc.

- WhatsApp :white_check_mark:
- whatsapp ❌ 
- Whatsapp ❌

**Pull request closed without merging**